.github/workflows/validate-data.yml

```
@@ -12,7 +12,9 @@ jobs:
         with:
           python-version: 3.6
       - name: Install packages
-        run: sudo apt-get install libxml2-utils apache2-dev libldap2-dev libsasl2-dev libyaml-dev
+        run: |
+          sudo apt-get update
```

```suggestion
          sudo apt-get update && \
```
or is this already run with set -e?

---

Yup, all GHA runs are! https://github.com/opensciencegrid/topology/runs/1385808985?check_suite_focus=true#step:4:3